thunderfish/powerspectrum.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+import matplotlib.mlab as ml
+
+def calc_psd(data, samplerate, fresolution):
```

Comment:
just name it psd !
Add all the arguments (with default values) that numpy.psd has and pass them on (in particular the window function).